packages/core/helper-plugin/lib/src/components/GenericInput/index.js

```diff
@@ -91,6 +91,23 @@ const GenericInput = ({
 
   switch (type) {
     case 'bool': {
+      // in case the default value is null, the default attribute.default doesn't exist
+      const isNullable = rest.attribute?.default === undefined;
```

Comment:
This works when using the input in the CM but the API is not really suited for other usage.
Can we just infer on the onClear prop?

Reply:
I'm not sure I understand what you mean when you say "Can we just infer on the onClear prop?" The button to clear the field should only be visible, if the field is actually `nullable` because otherwise there is nothing to clear. Do you have a better idea, of how I can find that out?

Reply:
Ok it's the nullable part that I did not get, can we just add a prop instead of using the attribute one?

The `attribute` is spread in the CM for the custom inputs otherwise the goal of this props is not to be used in the GenericInput as this input is used in other part of the codebase. So my suggestion is to use a specific prop like `isClearable` instead of using the `attribute` one. Then in the CM you'll use the `isClearable={attribute.default === null}`

Reply:
The problem is though, that we need to pass in the `clearLabel` and `onClear` callback: first to be able to translate the string and second to have a custom callback (which calls `onChange`). I don't see a way around that, unfortunately.

Don't we have the `defaultValue` of a field somewhere yet, which I could use?